devel/codemod/Portfile

```diff
@@ -25,6 +25,6 @@ checksums           rmd160  39b8461ab9e16c68a8605fb18a1e8b6c46242bd2 \
                     sha256  dc72368bc95fab74373f700a32682d12d308828215c973674c06d63da3494dbb \
                     size    17756
 
-python.versions     311
+python.versions     313
```

Comment:
since this is not a `py-<port>` typically we would use `python.default_version` here, would you mind changing to that?